$NetBSD: patch-configure,v 1.3 2025/05/15 15:44:41 adam Exp $

pcap_version is not portable.

External liblinear must be configured later.

--- configure.orig	2025-05-06 17:33:27.000000000 +0000
+++ configure
@@ -6274,9 +6274,10 @@ else
 /* end confdefs.h.  */
 
 #include <stdio.h>
-extern char pcap_version[];
+#include <pcap/pcap.h>
 int main() {
   int major, minor1, minor2;
+  const char *pcap_version = pcap_lib_version();
   sscanf(pcap_version,"%d.%d.%d", &major, &minor1, &minor2);
   if (major > 0)
     return 0;
@@ -6310,9 +6311,10 @@ else
 #ifdef HAVE_NET_BPF_H
 #include <net/bpf.h>
 #endif
-extern char pcap_version[];
+#include <pcap/pcap.h>
 int main() {
   int major, minor;
+  const char *pcap_version = pcap_lib_version(); 
   sscanf(pcap_version,"%d.%d", &major, &minor);
   if ((major == 1 && minor >= 1) || major > 1)
     return 0;
@@ -7196,7 +7198,6 @@ if test "${with_liblinear+set}" = set; t
   *)
     CPPFLAGS="-I$with_liblinear/include $CPPFLAGS"
     LDFLAGS="-L$with_liblinear/lib $LDFLAGS"
-    have_liblinear=yes
   ;;
   esac
 
